libcudacxx/include/cuda/functional

```diff
@@ -68,6 +68,8 @@
 #include <cuda/std/type_traits>
 #include <cuda/std/functional>
 #include <cuda/std/utility>
+#include <cuda/std/__algorithm/min.h>
+#include <cuda/std/__algorithm/max.h>
```

Comment:
question: why are these needed in functional?

Reply:
This is because of backwards compatibility. We had them in `functional` and we do not expose `__algorithm` yet